src/mock/data.ts

```diff
@@ -83,14 +87,15 @@ export const gjennomforinger: Gjennomforing[] = [
 		lopenr: 75634,
 		opprettetAr: 2022,
 		startDato: faker.date.past(),
-		sluttDato: faker.date.future(),
+		sluttDato: faker.date.past(),
```

Comment:
nit: Kan være en fordel å bruke `recent()` isteden for past, iom at backend filtrerer bort de som er avsluttet og over 2 uker gamle, så kan det hende folk blir litt forvirret over om det plutselig vises en eldre gjennomføring som er avsluttet

Reply:
Lurt! Skal endre :)